owrx/source/perseussdr.py

```diff
@@ -0,0 +1,31 @@
+from .direct import DirectSource
+from owrx.command import Flag, Option
+
+
+#
+# perseustest -s 768000 -u 0 -f 14150000 -r-|csdr convert_s8_f|nmux --bufsize 192512 --bufcnt 260 --port 35989 --address 127.0.0.1
+# perseustest -a -t0    -o -
+# perseustest -d 9 -a -t 100000 -o -  -s 768000 -u 0 -f 14150000 -r-
+
+class PerseussdrSource(DirectSource):
+    def getCommandMapper(self):
+        return super().getCommandMapper().setBase("perseustest -p -d -1 -a -t 100000 -o -  ").setMappings(
+            {
+                "samp_rate": Option("-s"),
+                "tuner_freq": Option("-f"),
+                "rf_gain": Option("-u"),
+                "lna_gain": Option("-g"),
+                "rf_amp": Option("-x"),
+            }
+        ).setStatic("-r-")
```

Comment:
does the `setStatic("-r-")` do anything? can't find that as a documented flag, and it looks like it originally came from hackrf.py ;)

Reply:
In fact, it is :-)
What does SetStatic  mean ?

Reply:
it's just a way to have a fixed parameter at the end. it was necessary for hackrf because it needs to be at the end in that case. not really the best solution but works for now.